pkgs/development/libraries/at-spi2-core/default.nix

```diff
@@ -30,11 +32,13 @@ stdenv.mkDerivation rec {
   };
 
   nativeBuildInputs = [
+    glib
     meson
     ninja
     pkg-config
-    gobject-introspection
     makeWrapper
+  ] ++ lib.optionals withIntrospection [
+    gobject-introspection
   ];
 
```

Comment:
there's a hardcoded `    "-Dintrospection=yes"` below

Reply:
at-spi2-core will only introspection if it detects gobject-introspection, and the "introspection" option is not set to no (default is auto).  So we should probably just drop the `-Dintrospection=yes`.  I'll do that.